lua/plugins/neo-tree.lua

```diff
@@ -74,6 +83,8 @@ return {
           O = "system_open",
           h = "parent_or_close",
           l = "child_or_open",
+          Y = "copy_file_name",
+          ["<C-y>"] = "copy_absolute_path",
```

Comment:
Update to use the new simplified command names

```suggestion
          Y = "copy_name",
          ["<C-y>"] = "copy_path",
```